tests/view_tests/urls.py

```diff
@@ -104,3 +104,7 @@
     ),
     url(r'^render_no_template/$', views.render_no_template, name='render_no_template'),
 ]
+
+urlpatterns += [
```

Comment:
Gotcha. Could you add something to the test so that it fails if they're removed to prevent them from being inadvertently removed?